packages/catalog-process/test/db.test.ts

```diff
@@ -113,6 +111,8 @@ describe("database test", async () => {
       useSSL: config.eventStoreDbUseSSL,
     });
     catalogService = catalogServiceBuilder(postgresDB, readModelService);
+
+    catalogService = catalogServiceBuilder(postgresDB, readModelService);
```

Comment:
```suggestion
```
This is duplicated

Reply:
Removed, sorry. Probably I got confused during the merge.